### DISCUSSION TITLE
Unable to connect to presto using SSL

### DISCUSSION THREAD

**Discussion by snycc15** [apache/superset#17254] [Q&A / Help]
Generally, standard SSL needs to provide the following information: SSL = true, keystore = path / key, keystore_ password；
The JDBC string provides the user / password. I don't understand how spuerset asks for it.

**Comment by srinify** [comment]
Hi @snycc15 the EXTRAS field when you connect to a db in Superset is probably your best bet:

https://superset.apache.org/docs/databases/extra-settings#ssl-access-to-databases

See if this works for you:

https://superuser.com/questions/1450124/importing-presto-pyhive-certificates-to-apache-superset
